universal-application-tool-0.0.1/app/services/program/ExportDefinition.java

```diff
@@ -0,0 +1,35 @@
+package services.program;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.google.auto.value.AutoValue;
+import javax.annotation.Nullable;
+
+@AutoValue
+@JsonDeserialize(builder = AutoValue_ExportDefinition.Builder.class)
+public abstract class ExportDefinition {
+  @JsonProperty("engine")
+  /** Which export engine to use. */
+  public abstract String engine();
```

Comment:
(non-blocking) can we use an Enum for the engine strings?

Reply:
Done.